src/server/game/Movement/MovementGenerators/RandomMovementGenerator.cpp

```diff
@@ -168,8 +169,17 @@ void RandomMovementGenerator<Creature>::SetRandomLocation(Creature* owner)
     Movement::MoveSplineInit init(owner);
     init.MovebyPath(_path->GetPath());
     init.SetWalk(walk);
-    time_t traveltime = init.Launch();
-    _timer.Reset(traveltime + resetTimer);
+    int32 splineDuration = init.Launch();
+
+    _wanderSteps--;
```

Comment:
nitpicking master attacks again 💃 

by default its better to use inline modifying operators, or w/e the name is, as a pre-operation
I'm making up all these names, I know

I mean ```--_wanderSteps;``` is better than ```_wanderSteps--;```

Reply:
https://en.cppreference.com/w/cpp/language/operator_incdec

meh, I didnt go too far from the real term 💃

Reply:
> Because a temporary copy of the object is constructed during post-increment and post-decrement, pre-increment or pre-decrement operators are usually more efficient in contexts where the returned value is not used.

There, that's why